launch/view_ur.launch.py

```diff
@@ -153,10 +169,6 @@ def generate_launch_description():
         arguments=["-d", rviz_config_file],
     )
 
-    nodes_to_start = [
-        joint_state_publisher_node,
-        robot_state_publisher_node,
-        rviz_node,
-    ]
+    nodes_to_start = [joint_state_publisher_node, robot_state_publisher_node, rviz_node]
```

Comment:
I would prefer to keep this "multi-row" because it is easier to read and add nodes if needed. I expect people to copy-paste and extend this file.